.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,42 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: Chromatic Publish and Testing
+
+# Event for the workflow
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+    paths:
+      # Only run on file changes in any of these paths
+      - "src/components/**/*"
+      - "src/pages/**/*"
+      - "src/pages-conditional/**/*"
+      - "src/templates/**/*"
```

Comment:
Only trigger in a PR that is opened, has new commits or related changes, or is converted from draft to "Ready for Review".

Also only runs if any files in the given paths have changed.

Reply:
@pettinarip I should probably add the path to the theme config to allow for review of visual changes if any values are altered. What do you think?

Reply:
Right! just seeing this now, sorry. I agree with that. Will add it.